guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>For more information about the API, visit the <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.toc.html#mmdetection" target="_blank">API reference for MMDetection</a>. For a detailed object detection workflow, refer to a sample <a href="https://developers.arcgis.com/python/sample-notebooks/detecting-and-categorizing-brick-kilns-from-satellite-imagery/" target="_blank">notebook</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>